modules/custom/az_event/az_event.module

```diff
@@ -51,6 +51,18 @@ function az_event_entity_extra_field_info() {
   return $extra;
 }
 
+/**
+ * Implements hook_preprocess_field().
+ */
+function az_event_preprocess_field(&$variables, $hook) {
+  $element = $variables['element'];
+  if ($element['#field_name'] === 'field_az_event_category') {
+    foreach ($variables['items'] as $key => $item) {
+      $variables['items'][$key]['content']['#options']['attributes']['class'][] = 'badge badge-link badge-light float-left mr-2';
```

Comment:
Recommend adding the `mb-2` class so there is space between them if they wrap. The `float-left` class ends up overriding this feature of the `badge` class.
```suggestion
      $variables['items'][$key]['content']['#options']['attributes']['class'][] = 'badge badge-link badge-light float-left mr-2 mb-2';
```